src/Symfony/Component/Process/Process.php

```diff
@@ -1671,13 +1671,8 @@ private function replacePlaceholders(string $commandline, array $env)
 
     private function getDefaultEnv(): array
     {
-        $env = [];
-
-        foreach ($_SERVER as $k => $v) {
-            if (\is_string($v) && false !== $v = getenv($k)) {
```

Comment:
Removal of the is_string check seems to be allowing arrays to get in to the env array as values. This causes an error when the env vars are read as a string in the start() method()

Reply:
Already fixed, see https://github.com/symfony/symfony/pull/44208
Please create issue instead of commenting on closed PRs to report issue btw.